de/README.md

```diff
@@ -3,19 +3,19 @@ wallabag Dokumentation
 
 ![wallabag Logo](../img/wallabag.png)
 
-**wallabag** ist eine Read-it-later Applikation: es speichert Websites,
-indem es nur den Inhalt behält. Elemente wie Navigation oder Werbung
+**wallabag** ist eine Read-it-later Applikation: sie speichert Websites,
+indem nur deren Inhalte behalten werden. Elemente wie die Navigation oder Werbung
 werden gelöscht.
 
-Die Hauptdokumentation für diese Applikation ist in einigen Abschnitten
+Die Hauptdokumentation für diese Applikation ist in mehreren Abschnitten
 organisiert:
```

Comment:
Ich bevorzuge ja "ist in mehrere Abschnitte gegliedert:". Klingt irgendwie harmonischer 😄